Framework/MDAlgorithms/src/IntegrateQLabEvents.cpp

```diff
@@ -240,13 +247,19 @@ PeakShapeEllipsoid_const_sptr IntegrateQLabEvents::ellipseIntegrateEvents(
     }
   }
 
+  double sqrtSigmas[3];
```

Comment:
The sigmas vector is the sqrt of the variances (eigenvalues) determined from diagonalizing the covariance matrix (see L96 in this file). I might be missing something, but I believe we want the radius to be ~3*sigma along each direction so why are we scaling the radii by the sqrt(sigma)?

Reply:
It seems like I misunderstood the process here. I thought sigma is the eigenvalue, which prompts me to use the sqrt here. Do you have some reference that is relevant to the math here?  I would like to learn a little bit more about it. Thanks

Reply:
So on L93 of this file we have (sorry don't know how to do the github link)
```
  std::vector<double> sigmas(3);
  for (int i = 0; i < 3; i++)
    sigmas[i] = sqrt(eigen_values[i]);
```
the eigenvalues of the covariance matrix are the variances along the axes of the ellipsoid (eigenvectors). The standard deviation, sigma = sqrt(variance) - assuming Gaussian distribution. So to include ~99.73% of events (if was Gaussian) we want to have a radius of 3*sigma[i] .This means the ratio of the radii is given by the ratio of sigma (i.e. the standard deviation) - not sqrt(sigma).

Reply:
On a second pass of the code, I see the mistake I made here.  I will revert the changes here :sweat:

Reply:
Great, thanks! Also in future perhaps our default choice of container should be a vector (automatically deleted when out of scope etc.)

Reply:
@RichardWaiteSTFC I had completely missed L93. I thought `sigmas[]` were the eigenvalues themselves. Thanks for spotting this! 😄